index.html

```diff
@@ -1400,9 +1400,34 @@ <h2>Core Vocabulary Definitions</h2>
 <p>
      <span class="rfc2119-assertion" id="td-context-default-language">
      In the <code>@context</code> array, the optional default language that is 
-     used throughout a TD document is defined within an object as a member with 
-     a key <code>@language</code> whose value MUST be a string representing a [[!BCP47]] 
-     language code.</span>
+     used throughout a TD document for human-readable strings outside of a <a href="#multilanguage">MultiLanguage</a> object
+     is also defined as a name 
+     <code>@language</code> whose value MUST be language tag from [[!BCP47]], 
+     possibly including a script subtag.</span>
+</p>
+<p>
+     Identification of the base direction of human-readable text in TDs 
+     is based on the following set of rules:
+     <ul>
+     <li><span class="rfc2119-assertion" id="td-context-default-language-direction-inference">
+     The base text direction MUST be inferred from the default language.
```

Comment:
Well... this is for texts outside of MultiLanguage, as stated in the introductory paragraph.   However, it is true that this assertion is not standalone.   Also, if we add my supportLanguage extension, there will be another exception.   However, I can replace this with your sentence (and extend it again in my extended PR for supportLanguage...)

Reply:
I changed this to
"Outside of  MultiLanguage instances the base text direction MUST be inferred from the default language tag."
and added the clarifying text after the list of rules:
"Rules inside MultiLanguage instances are similar but use the language tags given inside such instances instead of the default language tag."
Later, under MultiLanguage, another set of assertions is given.